src/sql/workbench/contrib/notebook/browser/cellViews/codeCell.component.ts

```diff
@@ -133,4 +129,12 @@ export class CodeCellComponent extends CellView implements OnInit, OnChanges {
 	public cellGuid(): string {
 		return this.cellModel.cellGuid;
 	}
+
+	public onKey(e: KeyboardEvent) {
```

Comment:
I don't see this being called anywhere - is this something from a base class?

Reply:
And why is this change here anyways? Doesn't seem related to your changes

Reply:
Both textCell and codeCell have this handler. We had a conflict where hitting Esc key while modal is open would switch the textCell edit mode to inactive AND hide the modal. That was an unintended side-effect so the decision was made to add this to both cell types.